_tests/util_test.go

```diff
@@ -0,0 +1,16 @@
+package _tests_test
+
+import (
+	"fmt"
+	"math/rand"
+
+	"github.com/onsi/ginkgo/config"
+)
+
+func init() {
```

Comment:
I started putting this type of common code in the test suite file itself, `tests_suite_test.go`. Is there a logical split between these funcs and those? Otherwise we should probably keep them together in one or the other file.